<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefineMapMenu</class>
 <widget class="QMainWindow" name="DefineMapMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SET UP DISPLAY</string>
    </property>
   </widget>
   <widget class="QLabel" name="directionSurveyLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Survey</string>
    </property>
   </widget>
   <widget class="QLabel" name="LLDisplayLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Left corner of display:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LLxDisplayLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>X : </string>
    </property>
   </widget>
   <widget class="QLabel" name="LLyDisplayLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Y : </string>
    </property>
   </widget>
   <widget class="QLabel" name="URyDisplayLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Y : </string>
    </property>
   </widget>
   <widget class="QLabel" name="URxDisplayLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>X : </string>
    </property>
   </widget>
   <widget class="QLabel" name="URDisplayLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Right corner of display:</string>
    </property>
   </widget>
   <widget class="QLabel" name="parallelDisplayLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Survey parallel to </string>
    </property>
   </widget>
   <widget class="QLabel" name="parallelAxisDisplayLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLineDisplayLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Line spacing : </string>
    </property>
   </widget>
   <widget class="QLabel" name="spaceMarkDisplayLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark spacing : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="doneEnteringInfoButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DONE ENTERING INFO</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLabel2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="TimeLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>1725650274</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLabel4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>550</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLabel3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLabel5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>450</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLabel7">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>450</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLabel6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>490</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spaceLabel8">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>490</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLabel</class>
   <extends>QLabel</extends>
   <header>timelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
